gtsfm/frontend/verifier/dummy_verifier.py

```diff
@@ -18,27 +18,30 @@ class DummyVerifier(VerifierBase):
     """A dummy verifier which produces random results"""
 
     def __init__(self):
-        super().__init__(min_pts=8)
+        super().__init__(min_pts_e_matrix=5, min_pts_f_matrix=8)
```

Comment:
do we still need the dummy verifier, or can we remove it now to keep the codebase lean?

Reply:
removed